pelicanconf.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import unicode_literals
+
+AUTHOR = u'Pyninsula Organizers'
```

Comment:
...though if you're going to use `unicode_literals` you can drop the `u`s, of course.